Sources/Core/GatekeeperList.swift

```diff
@@ -0,0 +1,25 @@
+// Copyright (c) 2014-present, Facebook, Inc. All rights reserved.
+//
+// You are hereby granted a non-exclusive, worldwide, royalty-free license to use,
+// copy, modify, and distribute this software in source code or binary form for use
+// in connection with the web services and APIs provided by Facebook.
+//
+// As with any software that integrates with the Facebook platform, your use of
+// this software is subject to the Facebook Developer Principles and Policies
+// [http://developers.facebook.com/policy/]. This copyright notice shall be
+// included in all copies or substantial portions of the software.
+//
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+// IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS
+// FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR
+// COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER
+// IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN
+// CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+
+import Foundation
+
+struct RemoteGatekeeperList: Decodable {
```

Comment:
Not necessary

Reply:
I looked at this again and think it's helpful to have this as it's own object. Here's why.
To have the equivalent with typealias would look like this:

```swift
typealias GatekeeperList = [String: [Gatekeeper]]
typealias RemoteGatekeeperList = [String: [GatekeeperList]]

extension RemoteGatekeeperList {
  var gatekeepers: [Gatekeeper] {
    guard let list = self["data"]?.first?["gatekeepers"] else {
      return []
    }
    return list
  }
}

```

Not terrible but in terms of decoding it's a lot less secure since it will decode any dictionary as a valid RemoteGatekeeperList. I'll change it if you feel strongly about it one way or the other. Not an issue, just let me know. 👍